dashbuilder/dashbuilder-shared/dashbuilder-js/packages/time-series-component/static/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Hello @manaswinidas 

Please add the category "Reporting" here so it will show up under "Reporting" in Components pallet.